### PR TITLE
test(e2e):input input group

### PR DESCRIPTION
**Pull request by AntonOlkhovskyi**
#### Please provide a link to the associated issue.
https://github.com/SAP/fundamental-ngx/issues/3732
#### Please provide a brief summary of this pull request.
Add e2e test input and input-group components
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: e2e/test-accessibility.e2e-spec.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+describe('Test run accessibility', function() {
+    it('should audit button page', async ()=> {
+        await browser.get('https://sap.github.io/fundamental-ngx/#/core/button');
+        runAxeTest('Test accessibility', '#page-content');
+        expect(false).toBe(false);
+    });
+});
```
Comment (Seamoo13) [inline e2e/test-accessibility.e2e-spec.js:7]: Do we need to have this now?

File: e2e/wdio/driver/wdio.ts [hunk 2]
```diff
@@ -68,6 +68,10 @@ export class Wdio {
         return $$(selector)[index].getValue();
     };
 
+    getValueArr(selector: string, sliceStart?: number, sliceEnd?: number): string[] {
```
Comment (Seamoo13) [inline e2e/wdio/driver/wdio.ts:71]: maybe "getArrValues" is a better name?

File: e2e/wdio/driver/wdio.ts [hunk 3]
```diff
@@ -198,6 +202,10 @@ export class Wdio {
 
     }
 
+    isExists(selector: string): boolean {
```
Comment (Seamoo13) [inline e2e/wdio/driver/wdio.ts:205]: maybe "doesItExist" is a more readable name? or 'doesElementExist'

File: e2e/wdio/platform/fixtures/appData/input-group-page-contents.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
+export default {
+    standartInputLabels: [
```
Comment (Seamoo13) [inline e2e/wdio/platform/fixtures/appData/input-group-page-contents.ts:2]: spelling correction: standard, not standart (same for below)

File: e2e/wdio/platform/pages/input.po.ts [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+export class InputPo extends BaseComponentPo {
+    readonly url = '/input';
+    readonly root = '#page-content';
+
+    defaultInput = '#input1';
+    textInput = '#input2';
+    numberInput = '#input3';
+    compactInput = '#input4';
+    readonlyInput = '#input5';
+    disabledInput = '#input6';
```
Comment (Seamoo13) [inline e2e/wdio/platform/pages/input.po.ts:13]: is there any way not to use the random-gen IDs? or are you sure these will be stable?

File: e2e/wdio/platform/tests/input-group.e2e-spec.ts [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+            webDriver.scrollIntoView(inputGroupPage.withFormInput);
+            webDriver.mouseHoverElement(inputGroupPage.withFormInputQuestionMark);
+
+            expect(webDriver.getText(inputGroupPage.withFormInputInfoTooltip)).toBe('This is tooltip to help');
+           // return;
+       // }
+    });
+
+    // TODO Unxit after merge
+    xit('should check RTL', () => {
```
Comment (Seamoo13) [inline e2e/wdio/platform/tests/input-group.e2e-spec.ts:142]: Why skipped?  I think you could add in the same manner as all the other tests, yes?

File: e2e/wdio/platform/tests/input.e2e-spec.ts [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+        webDriver.waitElementToBePresentInDOM(inputPage.textInput);
+        webDriver.waitForDisplayed(inputPage.textInput);
+        webDriver.addValue(inputPage.textInput, inputTestData.number);
+        webDriver.addValue(inputPage.textInput, inputTestData.special_characters);
+        webDriver.addValue(inputPage.textInput, inputTestData.text);
+
+        expect(webDriver.getValue(inputPage.textInput))
+            .toEqual(inputTestData.number + inputTestData.special_characters + inputTestData.text); // ???
+    });
+    // TODO: it is not working the same for manual and automation.
```
Comment (Seamoo13) [inline e2e/wdio/platform/tests/input.e2e-spec.ts:57]: Did you already create an issue? If so, please comment the issue # here so we will be able to check it later on.

File: e2e/wdio/platform/tests/input.e2e-spec.ts [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+        expect(webDriver.isEnabled(inputPage.disabledInput)).toBe(false);
+    });
+
+    it('have placeholder', () => {
+        expect(webDriver.getAttributeByNameArr(inputPage.inputsArray, 'placeholder', 1))
+            .toEqual(inputPContent.placeholdersArray);
+    });
+
+    it('should have error border color', () => {
+        if (browser.capabilities.browserName === 'Safari' || browser.capabilities.browserName === 'internet explorer') {
```
Comment (Seamoo13) [inline e2e/wdio/platform/tests/input.e2e-spec.ts:105]: I think we can update these with the new methods: webdriver.browerIsIEorSafari()

File: e2e/wdio/platform/tests/input.e2e-spec.ts [hunk 9]
```diff
@@ -0,0 +1,158 @@
...
+    });
+
+    it('should compact be smaller than the default', () => {
+        const defaultHeight = webDriver.getElementSize(inputPage.defaultInput) as WebdriverIO.SizeReturn;
+        const compactHeight = webDriver.getElementSize(inputPage.compactInput) as WebdriverIO.SizeReturn;
+
+        expect(defaultHeight.height).toBeGreaterThan(compactHeight.height);
+    });
+    // TODO Unxit after merge
+    xit('should check RTL', () => {
```
Comment (Seamoo13) [inline e2e/wdio/platform/tests/input.e2e-spec.ts:155]: why skipped? have method on base-component for it)

**Review by Seamoo13 — CHANGES_REQUESTED** [review]

**Comment by AntonOlkhovskyi** [comment]
Close. Open PR from the SAP repo.

**Pull request closed without merging**